frontend/src/dialog/share.vue

```diff
@@ -100,7 +100,7 @@
           </v-expansion-panel-content>
         </v-expansion-panel>
 
-        <v-container fluid text-xs-left class="pb-0 pt-3 pr-0 pl-0 caption">
+        <v-container fluid :text-xs-left="!rtl" :text-xs-right="!rtl" class="pb-0 pt-3 pr-0 pl-0 caption">
```

Comment:
Looks like this got copy-pasted...
```suggestion
        <v-container fluid :text-xs-left="!rtl" :text-xs-right="rtl" class="pb-0 pt-3 pr-0 pl-0 caption">
```